### PR TITLE
 Added completed 5-th Task

### PR DESCRIPTION
**Pull request by Stcher**

### CODE REVIEW

**Comment by Stcher** [comment]
[5/14 Main Layout: Footer](https://trello.com/c/cLMDdZrF/120-5-14-main-layout-footer)

File: app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="head.components">
```
Comment (linvor) [inline app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml]: // need to remove all unnecessary

<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>
            <move element="form.subscribe" destination="main.content" as="new_form.subscribe" after="-" before="-"/>
    </body>
</page>

File: app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+            <block class="Magento\Framework\View\Element\Js\Components" name="newsletter_head_components" template="Magento_Newsletter::js/components.phtml" ifconfig="newsletter/general/active"/>
+        </referenceBlock>
+        <referenceContainer name="footer">
+            <block class="Magento\Newsletter\Block\Subscribe" name="form.subscribe" as="subscribe" before="-" template="Magento_Newsletter::subscribe.phtml" ifconfig="newsletter/general/active"/>
+        </referenceContainer>
+        <move element="form.subscribe" destination="main.content" as="new_form.subscribe" after="-" before="-"/>
```
Comment (linvor) [inline app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml]: after="-" before="-" ???

File: app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/web/css/source/_extend.less [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+    .newsletter {
+        display: flex;
+        justify-content: space-between;
+
+        .brand-title {
+            margin-left: 80px;
+
+            h2 {
+                margin-bottom: 0;
+                font-family: 'Poppins',sans-serif;
```
Comment (linvor) [inline app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/web/css/source/_extend.less]: Use a variable (@font-family__base)

Comment (Stcher) [reply app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/web/css/source/_extend.less]: Done, also I created new variables for custom fonts and replace it in the whole project

File: app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml (hunk 2 above)
Comment (Stcher) [reply app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml]: Done, remove it due to no needs

File: app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml (hunk 1 above)
Comment (Stcher) [reply app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml]: Done

**Pull request merged**